<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Будильник</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Будильник</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Время проигрывания</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="alarmTime">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>118</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбор песни</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="urlLine">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="fromFileButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Из файла...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="setAlarmClock">
        <property name="text">
         <string>Завести будильник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Таймер</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>351</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Время для отдыха</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeForBreak">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбор песни</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="urlLine_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="fromFileButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Из файла...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Время для работы</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="userTime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Таймер закончит работу</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="finalTime">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="setTimerClock">
        <property name="text">
         <string>Завести будильник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_2">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
